packages/web-app-admin-settings/src/components/Spaces/SideBar/MembersPanel.vue

```diff
@@ -1,103 +1,115 @@
 <template>
   <div class="oc-ml-s">
-    <div v-if="spaceResource.spaceRoles.manager.length" class="oc-mb-m">
-      <h4 v-translate>Managers</h4>
-      <div
-        v-for="(manager, index) in [...spaceResource.spaceRoles.manager].sort((a, b) =>
-          a.displayName.localeCompare(b.displayName)
-        )"
-        :key="index"
-        class="oc-flex oc-flex-middle oc-mb-s"
-      >
-        <oc-avatar
-          v-if="manager.kind === 'user'"
-          :user-name="manager.displayName"
-          :width="36"
-          class="oc-mr-s"
-        /><oc-avatar-item
-          v-else
-          :width="36"
-          icon-size="medium"
-          :icon="groupIcon"
-          name="group"
-          class="oc-mr-s"
-        />
-        {{ manager.displayName }}
+    <oc-text-input
+      v-model="filterTerm"
+      class="oc-text-truncate oc-mr-s oc-mt-m"
+      label=""
+      :placeholder="$gettext('Filter members')"
+    />
+    <div ref="membersListRef">
+      <div v-if="!filteredSpaceMembers.length">
+        <h4 v-text="$gettext('No members found')" />
```

Comment:
All the `h4` tags here need to be `h3`s, otherwise the hierarchy is broken. The panel title is a `h2` and there are not `h3` in between.